### DISCUSSION TITLE
How many RAM resources do you usually use?

### DISCUSSION THREAD

**Discussion by sunghochoi122** [seominjoon/kaist-ai605#114] [Q&A] [answered]
How many RAM resources do you usually use?
Just computing word counts for all documents consume more than half of the maximum RAM resource.
The word counts for each document is stored in a dictionary, e.g., {0: 0, 1: 0, 2: 1, ...}

Being terminated, the runtime does not free up the memory, so the runtime crashes after or during one or two times of running.

Am I using the memory inefficiently, or am I not the only one who has this issue?

**Comment by sorobedio** [comment, marked as answer]
if you try to use training context to create the word count you will have this problem. instead i use training context only for vocab and word2idx and discard it then use the validation context for word count for all documents. i don't know if it is the right way but that is how i avoided the memory problem

**Reply by sunghochoi122** [reply]
Thanks for your answer. I just end up using validation data for vocab and reimplementing everything.
